index.html

```diff
@@ -1,81 +1,81 @@
 <!DOCTYPE html>
-<html lang = en>
-<head>
-	<meta charset="utf-8">
-	<title>Rocket Science</title>
-</head>
-<body>
-	<div id = "branding">
-		<img src = "" alt = "logo"/>
-		<h1>Rocket Science</h1>
-		<h2>Software Craftsmanship</h2>
-	</div>
+<html lang=en>
+  <head>
+    <meta charset="utf-8">
+    <title>Rocket Science</title>
+  </head>
+  <body>
+    <div id="branding">
+      <img src="" alt="logo"/>
+      <h1>Rocket Science</h1>
+      <h2>Software Craftsmanship</h2>
+    </div>
 
-	<div id = "introText">
-		<p>Lorem ipsum dolor sit amet, quo eu sanctus omnesque, an quas splendide pro, maiorum fabellas per et. Vix et prima bonorum adipiscing, ius zril inermis ex. Sea ne omnis modus tincidunt, eos an cetero singulis torquatos. Et erat pertinacia cum, instructior reprehendunt vel ut. Ex eligendi insolens sapientem per, duo id tale sumo propriae, ut falli inermis cum.</p>
-	</div>
+    <div id="introText">
```

Comment:
we'll use camel case in javascript quite a bit - but for raw HTML and CSS i prefer to use the form `intro-text` and `main-content`. Old browsers didn't distinguish between upper- and lower-case characters and as a result it has become standard to use dashes in classnames!

Reply:
okay cool beans! commenting to show I'm learning how this all works! :)